src/components/TestSuiteRun/TestSuiteRunSummary.jsx

```diff
@@ -245,12 +264,12 @@ const mapStateToProps = (state) => {
 };
 
 const mapDispatchToProps = {
-  setCurrentTestRun,
+  setCurrentTestRunAction,
```

Comment:
should be `setCurrentTestRun: setCurrentTestRunAction`, this should fix the selections.

Reply:
Updated, thanks for tracking that down!